x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -518,6 +519,28 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
         },
       }),
   },
+  registered_domain: {
+    FieldsComponent: RegisteredDomain,
+    docLinkPath: '/registered-domain-processor.html',
+    label: i18n.translate('xpack.ingestPipelines.processors.label.registeredDomain', {
+      defaultMessage: 'Registered domain',
+    }),
+    typeDescription: i18n.translate(
+      'xpack.ingestPipelines.processors.description.registeredDomain',
+      {
+        defaultMessage:
+          'Extracts the registered domain (also known as the effective top-level domain or eTLD), sub-domain, and top-level domain from a fully qualified domain name.',
```

Comment:
```suggestion
          'Extracts the registered domain (effective top-level domain), sub-domain, and top-level domain from a fully qualified domain name.',
```

Reply:
I think we can shorten this a bit and still maintain context.

Reply:
Thanks for checking it out Adam!

Patched with https://github.com/elastic/kibana/pull/99643/commits/02558f493f7486df80e8db5c153e318ef93914e4